deploy/crds/toolchain_v1alpha1_useraccount.yaml

```diff
@@ -94,7 +72,7 @@ spec:
           required:
           - userID
           - nsLimit
-          - nsTemplateSet
```

Comment:
`nsTemplateName` should be required, shouldn't it?

Reply:
UserAccount is created by MUR_ctrl .. I made this `nsTemplateName` optional so MUR_ctrl no need to create anything related to NSTemplateSet and creation of NSTemplateSet is totally handler by UserAccountCtrl.